Sources/PayPalMessages/Logger/Logger.swift

```diff
@@ -1,5 +1,26 @@
 import Foundation
 
+struct CloudEvent: Encodable {
+
+    let specversion: String = "1.0"
+    let id: String
+    let type: String = "com.paypal.credit.upstream-presentment.v1"
+    let source: String = "urn:paypal:event-src:v1:ios:messages"
+    let datacontenttype: String = "application/json"
+    // swiftlint:disable:next line_length
+    let dataschema: String = "ppaas:events.credit.FinancingPresentmentAsyncAPISpecification/v1/schema/json/credit_upstream_presentment_event.json"
+    let time: String
+    let data: Logger
```

Comment:
Since this doesn't actually use any parts of the `Logger` itself aside from holding onto it, we should consider changing this to better represent how it's used with the generic `Encodable` type which will also decouple it a bit.